### PR TITLE
Allow setting a custom token lifetime

### PR DESCRIPTION
**Pull request by alberto**

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -142,3 +142,9 @@ EOF
     error_message = "The vpc_endpoints valid values are s3, sts and kinesis-streams."
   }
 }
+
+variable "token_lifetime_seconds" {
+  description = "(Optional) Token expiry lifetime. By default it is 2592000 (30 days)."
+  type = number
+  default = 2592000
```
Comment (XaviPolaNextail) [inline variables.tf:148]: wouldn't be better to make it `-1` (no expiry) by default?

**Review by XaviPolaNextail — APPROVED** [review]
LGTM, just a doubt regarding the default value.

File: variables.tf (hunk 1 above)
Comment (alberto) [reply variables.tf:148]: I have used the same default Databricks currently uses in the underlying module.

**Review by ferminho — APPROVED** [review]
:mechanical_arm:

**Pull request merged**